cmd/ncproxy/buckets.go

```diff
@@ -0,0 +1,51 @@
+package main
+
+import (
+	bolt "go.etcd.io/bbolt"
+)
+
+var (
```

Comment:
`const` instead?

Reply:
byte slices are not constants. I'm gonna move the schemaVersion to const though.